woocommerce/woocommerce-ios
# Fixed launch screen stretched

Fixes #1932 

## Description
This PR fixes the launch screen which appears stretched, probably caused by this PR https://github.com/woocommerce/woocommerce-ios/pull/1855.
I adapted the `UIImageView` inside `LaunchScreen.storyboard` to match the aspect ratio of the new logo image.

## Testing 
1) Delete the app on your device/simulator (to make sure that is not cached)
2) Run the app on different simulators.
3) Make sure that the launch screen is no more stretched.


## Screenshots

| Before             |  After |
:-------------------------:|:-------------------------:
![76358859-320f0000-62e8-11ea-95da-33f6ff052bfa](https://user-images.githubusercontent.com/495617/76430790-0277fa80-63b1-11ea-9ce9-bc659595c3e7.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-03-11 at 15 55 42](https://user-images.githubusercontent.com/495617/76430594-c775c700-63b0-11ea-9ddd-eb09de9e0d16.png)


Update release notes:
